app/channels/notifications_channel.rb

```diff
@@ -0,0 +1,12 @@
+class NotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_for current_user
+  end
+
+  def send_notification(data)
+    user = User.find(data['user_id'])
+    NotificationsChannel.broadcast_to(user,
```

Comment:
In addition to the changes requested in a previous comment, the layout should be:
```
NotificationsChannel.broadcast_to(
  user,
  new_match_with: '',
  new_matches_count: '1'
)